modules/videoio/src/cap_mfx_common.hpp

```diff
@@ -302,7 +303,25 @@ class VAHandle : public DeviceHandler {
     int file;
 };
 
-// TODO: Windows specific
+#endif
+
+// Windows specific
+#ifdef _MSC_VER
```

Comment:
The same problem: detection of platform via compiler is wrong.
`_WIN32` should be here.